test/integration/narrow/routes/endemics/endemics-package.test.js

```diff
@@ -14,17 +14,7 @@ describe('Endemics package test', () => {
 
   beforeAll(async () => {
     getEndemicsClaim.mockImplementation(() => { return { typeOfLivestock: 'pigs', reference: 'TEMP-6GSE-PIR8' } })
-    process.env.ENDEMICS_ENABLED = 'true'
-
-    jest.mock('../../../../../app/config', () => {
-      const originalModule = jest.requireActual('../../../../../app/config')
-      return {
-        ...originalModule,
-        endemics: {
-          enabled: true
-        }
-      }
-    })
+    jest.mock('../../../../../app/config')
```

Comment:
Do we need this? The original code was replacing the config with a copy of the original, whereas now we're just removing the entire thing.